src/block_conversion_plugin.cpp

```diff
@@ -129,7 +129,7 @@ class block_conversion_plugin_impl : std::enable_shared_from_this<block_conversi
          new_block.header.parent_hash = last_evm_block.header.hash();
          new_block.header.transactions_root = silkworm::kEmptyRoot;
          // Note: can be null
-         new_block.consensus_parameter_index = last_evm_block.consensus_parameter_index;
+         new_block.set_consensus_parameter_index(last_evm_block.get_consensus_parameter_index());
```

Comment:
This action will generate an empty (but not null) extra_block_data field in the block and got eventually serialized into the db i believe? will it cause some inconsistency?

Reply:
Will be changed in silkworm